getsentry/sentry-javascript
# Fix(integrations): Fix support for wildcard paths in react-router-dom v6 integration (#5997)

This PR attempts to address the issue described in #5997 where wildcard paths (`*`) in the react-router-dom v6 integration were not being properly reported on for transactions.

To test:
1. Pull this branch down
2. Clone [this repo](https://github.com/grahamhency/sentry-react-router-6-example) and follow the setup & usage instructions in the README.

I also added a test for this change. I noticed that the existing tests had odd react router route declarations as far as nesting goes so I updated those tests to reflect how nested routes within react router should be structured.
